processor/src/watcher.js

```diff
@@ -82,7 +79,7 @@ function watch(events, repoDir = '/repositories', checkIsRepoReady) {
   return unwatch
 }
 
-async function processRepo(events, repoDir, gitDir) {
+async function processRepo(repoDir, gitDir) {
   // /repositories/user/project.git -> user/project
   const name = path.relative(repoDir, gitDir).slice(0, -4)
```

Comment:
This your much shorter :laughing: of https://github.com/kitspace/kitspace-v2/blob/9499c57cc6824bc648e309a0f00ed54bad97f1b5/processor/src/checkIsRepoReady.js#L78-L88
I think we should use refactor this into a common utility to avoid updating it somewhere and forget the other one.

Reply:
Oh, nicely spotted.